### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by kghbln**
Provide German translations and shortcuts for days.

### CODE REVIEW

File: i18n/extra/de.json [hunk 1]
```diff
@@ -173,32 +173,32 @@
...
         [
-            "Tuesday",
-            "Tue"
+            "Dienstag",
+            "Di"
         ],
         [
-            "Wednesday",
-            "Wed"
+            "Mittwoch",
```
Comment (JeroenDeDauw) [inline i18n/extra/de.json]: Silly Germans, 3 is not the middle in [1, 7], that's 4 :)

Comment (kghbln) [inline i18n/extra/de.json]: So I read that you have learned your German lessons. :P

Comment (JeroenDeDauw) [inline i18n/extra/de.json]: nein

Comment (kghbln) [inline i18n/extra/de.json]: Hä, so you are not into learning esoteric languages. :D

**Pull request merged**